### ISSUE TITLE
Missing Dinos From List

### ISSUE THREAD

**Issue by James-Thamos** [coldino/Larkator#58]
Not sure if this is a known issue or not, but I feel that two dinos are missing that should be on the list to track.  The first is Alpha Rex.  The second is the Titanosaur. Don't know if this is a bug or intended thought it should be known though.  This is a great thing to have and I think you have done great work with it.  I will be continuing to use it regardless and recommending it to the people I know who play ark.

**Comment by coldino** [comment]
The list of species is simply showing what is present in your save file, with no filtering. So, if a species isn't listed then there are none currently spawned.

Or do you mean the Worries list? I guess that could be extended, but it was intended only as an example because it would be map or even player specific.

**Comment by James-Thamos** [comment]
I can't add it to either shopping or worried lists at all. I understand that when you click on them on the left if none show up they aren't spawned in, but I can't even add them to be tracked on either list?

**Comment by coldino** [comment]
Yeah, this isn't ideal. It only ever shows species found in your map, mostly so it will always support future Ark species, modded species, etc.

Perhaps I should add the list of known species names to this list, but it will bloat it significantly. I'll experiment.
